bitnami/harbor/Chart.yaml

```diff
@@ -31,4 +31,4 @@ maintainers:
 name: harbor
 sources:
   - https://github.com/bitnami/charts/tree/main/bitnami/harbor
-version: 16.6.5
+version: 16.6.6
```

Comment:
I think this change is a minor version. Could you please update the chart version?